internal/app/carrier/repository_test.go

```diff
@@ -0,0 +1,88 @@
+package carrier
+
+import (
+	"context"
+	"github.com/DATA-DOG/go-sqlmock"
+	"github.com/jmoiron/sqlx"
+	"github.com/lib/pq"
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/assert"
+	"parcel-service/internal/app/model"
+	"testing"
+	"time"
+)
+
+func TestRepository_UpdateCarrierRequest(t *testing.T) {
+	var sourceTime = time.Now()
```

Comment:
as we are using short form everywhere for this case, please use the same format here. 

sourceTime := time.Now()

Reply:
done